client/src/app/payroll/payroll.component.html

```diff
@@ -1 +1,34 @@
-<div class="table-margin"><app-payroll-list></app-payroll-list></div>
+<div class="ml-3 mr-3">
+  <div class="row full-height">
+    <div class="col-2 background-dark border-light">
+      <div class="row">
+        <div class="list-group-item list-group-item-action">
+          <div class="row">
+            <div class="col-12">
+              <span>
+                Departments
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        class="row"
+        *ngFor="let department of departments"
+        (click)="onSelectDepartment(department)"
+      >
+        <div class="list-group-item list-group-item-action">
+          <div class="row">
+            <div class="col-12">
+              <span>{{ department.replace('_', ' ') }}</span>
```

Comment:
This would make a great pipe since it's presentational.

```typescript
import { Pipe, PipeTransform } from '@angular/core';

@Pipe({name: 'unslugify'})
export class UnslugifyPipe implements PipeTransform {
  transform(value?: string): string {
    return value?.replace('_', ' ') ?? '';
  }
}
```

Register it in the `SharedModule` declarations and exports, a then use it like:

```html
<span>{{ department | unslugify }}</span>
```

Reply:
Nice! Almost as good as big daddy pipe!